framework/sac_rlhf.py

```diff
@@ -241,8 +254,21 @@ def get_action(self, x):
 
                 print("step", global_step, i)
 
+                SimTeacher = Teacher(
```

Comment:
Please rename to sim_teacher  as this describes a concrete instance of a teacher